Lib/test/pythoninfo.py

```diff
@@ -142,7 +142,7 @@ def collect_platform(info_add):
     info_add('platform.python_implementation',
              platform.python_implementation())
     info_add('platform.platform',
-             platform.platform(aliased=True))
+             platform.platform(aliased=True, terse=True))
```

Comment:
It seems like terse=True returns less information.

>>> platform.platform(aliased=True)
'Linux-4.16.14-300.fc28.x86_64-x86_64-with-fedora-28-Twenty_Eight'
>>> platform.platform(aliased=True,terse=True)
'Linux-4.16.14-300.fc28.x86_64-x86_64-with-glibc2.3.4'

Reply:
The distribution info. `platform.dist` is deprecated long ago and the buildbot already tells us what the exact distribution is. The info is already not available in master.